docs/source/llm/build-run-llama3-qualcomm-ai-engine-direct-backend.md

```diff
@@ -0,0 +1,128 @@
+# Building and Running Llama 3 8B Instruct with Qualcomm AI Engine Direct Backend
+
+This tutorial demonstrates how to export Llama 3 8B Instruct for Qualcomm AI Engine Direct Backend and running the model on a Qualcomm device.
+
+## Prerequisites
+
+- Set up your ExecuTorch repo and environment if you haven’t done so by following [the Setting up ExecuTorch](../getting-started-setup.md) to set up the repo and dev environment.
+- Read [the Building and Running ExecuTorch with Qualcomm AI Engine Direct Backend page](../build-run-qualcomm-ai-engine-direct-backend.md) to understand how to export and run a model with Qualcomm AI Engine Direct Backend on Qualcomm device.
+- Follow [the README for executorch llama](https://github.com/pytorch/executorch/tree/main/examples/models/llama2) to know how to run a llama model on mobile via ExecuTorch.
+- A Qualcomm device with 16GB RAM
+  - We are continuing to optimize our memory usage to ensure compatibility with lower memory devices.
+- The version of [Qualcomm AI Engine Direct SDK](https://developer.qualcomm.com/software/qualcomm-ai-engine-direct-sdk) is 2.25.0 or above.
```

Comment:
Might point to 2.26 version, or convolutions in you case (converted from linear with no bias) will fail to lower.

Reply:
Thanks for your reminder!